aten/src/ATen/native/ts_native_functions.yaml

```diff
@@ -102,7 +102,7 @@ full_codegen:
   - pow.Tensor_Scalar
   - pow.Tensor_Tensor
   - random
-  - random.from
+  - random.from_int
```

Comment:
Trying to follow, are we deleting one operator (`random.from`) and add an operator (`random.from_int`) in this pr?

Reply:
By browsing the previous pr, seems like we are trying to update `aten::random_.from`?

Reply:
changing `random_.from to random_.from_int` as noted in description

Reply:
I think upgrader is used for one operator, like changing schema or semantic. If it's renamed, we are doing two operations here:

1. delete one operator
2. add a new operator

I wonder which case will be easier for you

option 1. keep the same op name `random_.from` and change the schema (or semantic)
option 2. add a new operator `random_.from_int` and keep the original operator `random_.from` as is.

@tugsbayasgalan maybe can confirm.

Reply:
Why would doing two changes be different than doing one change at a time in two separate PRs ?